resources.html

```diff
@@ -85,7 +85,8 @@ <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
                 <div class="columns small-12 medium-12 large-5 align-self-bottom">
                     <nav class="floe-nav">
                         <ul>
-                          <li><a href="./news/index.html">News</a></li>
+                           <li><a href="./index.html">Home</a></li>
+                           <li><a href="./news/index.html">News</a></li>
```

Comment:
why making extra changes? also a PR is already present for this issue.

Reply:
I have already made the pr #155 for the same feature. Why again?

Reply:
actually i have not seen the pull request. I have seen the issue is still opened and reported in issue tab. It’s still not merged. Sorry for the inconvenience.

Reply:
> I have already made the pr #155 for the same feature. Why again?

that is why...it is necessary to give the issue number in your PR discription. So to link the issue with the pull request. So that anyone can see a PR has been made with respect to that issue.

Reply:
Okay thanks, will keep in mind this from next time.